src/main/bin/db/initPostgresSchema.sql

```diff
@@ -0,0 +1,172 @@
+('fjoa25pm', 'mtq2r09h', 'h9ib1a73'),
+('l06590op', 'mtq2r09h', 'u8489vqk'),
+('5gjnmbg9', 'b36as7qx', 'aa7hcumk'),
+('q50jleu2', 'b36as7qx', 'h9ib1a73'),
+('tvqkk4cl', '4b6v23zg', 'aa7hcumk'),
+('fhb9n1jw', '4b6v23zg', 'nmbyan9a'),
+('jdl3mgjq', 'csye07l8', 'nmbyan9a');
+
+DELETE FROM users;
+INSERT INTO users
+VALUES ('wx1of70l', 'harvey@utoronto.ca', 'Harvey', 'Donnelley', '6ca13d52ca70c883e0f0bb101e425a89e8624de51db2d2392593af6a84118090', current_timestamp), -- abc123
+('mtq2r09h', 'kento@utoronto.ca', 'Kento', 'Takeda', '8f61ad5cfa0c471c8cbf810ea285cb1e5f9c2c5e5e5e4f58a3229667703e1587', current_timestamp), -- def456
+('b36as7qx', 'paul@utoronto.ca', 'Paul', 'Bangalan', 'dd130a849d7b29e5541b05d2f7f86a4acd4f1ec598c1c9438783f56bc4f0ff80', current_timestamp), -- 123abc
+('4b6v23zg', 'darlyn@utoronto.ca', 'Darlyn', 'Nguyen', '5e689fe2eaed097f8a9dc74e6d2aaaef59688d820ecb4cefc7b1a65cc78fa0a2', current_timestamp), -- 456def
+('csye07l8', 'nyko@utoronto.ca', 'Nyko', 'Dionisio', '1ae86748124fc4bb2e8ddf430cf146689dec449c811096329beeef55d357967d', current_timestamp), -- 1abc2
+('hph52pue', 'nicholas@utoronto.ca', 'Nicholas', 'Mckee', '743d0b05d4b0b58b2b516b83919c782b0443e25e1b2618b28328cbae7570ff9a', current_timestamp), -- 3def4
+('bzjpq4bh', 'humraj@utoronto.ca', 'Humraj', 'Bhoday', '1f91bc2741ddcd07dc2abd277d412010e6e0bdf6532aae15e8d0715555415661', current_timestamp), -- 5ghi6
+('timlxndy', 'zhihao@utoronto.ca', 'Zhihao', 'Lim', '5a3cf14cf8263fa316a6dad08d1a2f14432fcbfd74e3ae97e9fd1b3dee16703b', current_timestamp); -- 7jkl8
+
+DELETE FROM tests;
+INSERT INTO tests 
+VALUES ('igt4sstc', 'wx1of70l', 'h9ib1a73', 'MCQ', '10', '30', current_timestamp),
+('wdgi1rq1', 'mtq2r09h', 'h9ib1a73', 'MCQ', '20', '60', current_timestamp),
+('86pmfyv6', 'wx1of70l', 'aa7hcumk', 'Multiselect', '10', '20', current_timestamp),
+('lfv2iub7', 'b36as7qx', 'koz8t694', 'Open-ended', '5', '60', current_timestamp),
+('54exqbk6', 'b36as7qx', 'nmbyan9a', 'MCQ', '30', '60', current_timestamp),
+('6uobpwih', '4b6v23zg', 'koz8t694', 'Multiselect', '10', '70', current_timestamp),
+('kv9nno9k', '4b6v23zg', 'aa7hcumk', 'MCQ', '20', '40', current_timestamp),
+('igptejsm', 'csye07l8', 'nmbyan9a', 'Open-ended', '3', '50', current_timestamp),
+('33bpyulj', 'hph52pue', 'h9ib1a73', 'MCQ', '1', '50', current_timestamp),
+('beeg1li4', 'bzjpq4bh', 'aa7hcumk', 'MCQ', '12', '25', current_timestamp);
+
+DELETE FROM solutions;
+INSERT INTO solutions
+VALUES ('r5hdyrg2', 'igt4sstc', 'mtq2r09h', 5, 10, 20, '5ogwodol', current_timestamp),
+('v5apbbr0', 'igt4sstc', 'b36as7qx', 10, 10, 20, 'iccijdyo', current_timestamp),
+('pw7mh6vf', 'wdgi1rq1', 'b36as7qx', 15, 10, 20, 'gso1dmii', current_timestamp),
+('hi0ve0z5', 'wdgi1rq1', 'mtq2r09h', 2, 10, 20, 'uo802k9i', current_timestamp),
+('wq3t8cps', '86pmfyv6', '4b6v23zg', 10, 10, 20, 'v5e3zcec', current_timestamp);
+
+DELETE FROM messages;
+INSERT INTO messages
+VALUES ('w2ilpab8', 'r5hdyrg2', 'b36as7qx', '5ogwodol', 'great answer!', current_timestamp),
```

Comment:
This is great ! A small issue is that the root message needs to exist in the messages table for us to construct the DiscussionBoard tree. For rows in the messages table corresponding to root messages, the parent_id can be left as null (this indicates that it is a root node).

Reply:
I've left the parent_id as null to indicate the root nodes.